stdlib/3/os/__init__.pyi

```diff
@@ -343,10 +344,16 @@ if sys.version_info >= (3, 3):
     def replace(src: _PathType, dst: _PathType) -> None: ...
 def rmdir(path: _PathType) -> None: ...
 if sys.version_info >= (3, 6):
+    class ScandirIterator(Iterable):
```

Comment:
This needs to be generic in AnyStr (so inherit from `Iterable[AnyStr]`, and then change all the `str` below to `AnyStr`.

Reply:
Also, the name should be prefixed with `_` because this name is not publicly exported in the `os` module.